lib/Dancer/Error.pm

```diff
@@ -165,7 +165,7 @@ sub _censor {
         if (ref $hash->{$key} eq 'HASH') {
             $censored += _censor( $hash->{$key}, $recursecount );
         }
-        elsif ($key =~ /(pass|card?num|pan|cvv|cvv2|ccv|secret|private_key)/i) {
+        elsif ($key =~ /(pass|card?num|pan|cvv2?|ccv|secret|private_key|cookie_key)/i) {
```

Comment:
Why the removal of the "2?" from that regex?  It's intended to match either `cvv` or `cvv2`, two names for "the last 3 digits on the signature strip" of a credit/debit card.

I wonder if this should just be a bit more generic and match anything with `_key`?   Or just include `api_key`?

Reply:
"2?" has not been removed - it's been added, as a simplification of cvv|cvv2.

"_key" is a bit of a giveaway that something might be security related, and I'd support expanding the paranoia to cover that.